### PR TITLE
Restructure top level links in docusaurus

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

File: plugin-development.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ Also include the following support information:
 |-----------------------|-----------|
 | Support               | Provide information about how to get help with the plugin or module. Link to the plugin's support system. Do not link to Vipps contact page. |
 | Link to Vipps Portal  | Inform the merchant that they can manage their sales units on the Vipps portal. For example: "You can manage your sales unit (e.g., change name and logo), see reports of all payments, etc. on [portal.vipps.no](https://portal.vipps.no)." |
-| Link to Vipps FAQ     | Provide a link to the Vipps FAQs. For example: "For typical questions and answers about Vipps payments and APIs, see the [Vipps FAQs](https://developer.vippsmobilepay.com/docs/vipps-developers/faqs)." |
```
Comment (okpedersen) [inline plugin-development.md:44]: Is this right?

File: plugin-development.md [hunk 2]
```diff
@@ -41,11 +41,11 @@ Also include the following support information:
...
+| Link to Vipps FAQ     | Provide a link to the Vipps FAQs. For example: "For typical questions and answers about Vipps payments and APIs, see the [Vipps FAQs](https://developer.vippsmobilepay.com/docs/
 | Info about contributing on GitHub | Add to the *issues* tab on the GitHub repo. |
 | Link to Vipps help pages | "[The Vipps help page](https://vipps.no/hjelp/vipps/)" |
 | Link to Vipps customer center for merchants | "If you need help with your customer relationship with Vipps, [Contact Vipps customer service](https://vipps.no/hjelp/vipps/)." |
-| Link to newsletters       | "Vipps has a [technical newsletter](https://developer.vippsmobilepay.com/docs/vipps-developers/newsletters) with news about APIs, etc. We recommend subscribing to it." |
```
Comment (okpedersen) [inline plugin-development.md:48]: Is this right?

File: plugin-development.md [hunk 3]
```diff
@@ -12,11 +12,11 @@ END_METADATA -->
 
 Follow these guidelines while creating plugin modules. These are defined to minimize manual support.
 
-If you are new to developing with the API, see the [Getting started](https://developer.vippsmobilepay.com/docs/vipps-developers/getting-started) guide.
```
Comment (okpedersen) [inline plugin-development.md:15]: Is this right?

File: plugin-development.md (hunk 2 above)
Comment (rebekaburnett) [reply plugin-development.md:48]: I redid everything and updated the history, so it's hard to see what I did, does it look ok now?

**Pull request merged**